lib/ret/entity_state.ex

```diff
@@ -0,0 +1,92 @@
+defmodule Ret.EntityState do
+  import Ecto.Query, warn: false
+  alias Ret.{Hub, Repo}
+
+  alias Ret.EntityState.{CreateMessage, UpdateMessage}
+
+  def list_entity_states(hub_id) do
```

Comment:
Please write tests for this function

Reply:
Yep, I will (re-)add tests after we get a chance to discuss whether the design is going in the right direction